src/autocomplete.js

```diff
@@ -8,18 +8,18 @@ import {cbToCb, composeEventHandlers, debounce, scrollIntoView} from './utils'
 class Autocomplete extends Component {
   static propTypes = {
     children: PropTypes.func.isRequired,
-    defaultSelectedItem: PropTypes.any,
     defaultHighlightedIndex: PropTypes.number,
+    defaultValue: PropTypes.any,
     getA11yStatusMessage: PropTypes.func,
     getValue: PropTypes.func,
+    multiple: PropTypes.bool,
     onChange: PropTypes.func.isRequired,
     onClick: PropTypes.func,
+    value: PropTypes.any,
   }
 
   static defaultProps = {
-    defaultSelectedItem: null,
```

Comment:
Should this be `defaultValue: null`? Even though it doesn't actually make a difference, I like having defaults set for anything that's not required.